### DISCUSSION TITLE
Who's responsible for 3.0.x.x_Maintenance, for future 3.0.3.x releases ?

### DISCUSSION THREAD

**Discussion by mhcwebdesign** [opencart/opencart#10685] [General]
Just asking, because there have been quite a few bugfixes done in the 3.0.x.x_Maintenance branch since OC 3.0.3.8 was released, and it might perhaps be useful to do a new OC 3.0.3.9 release off the 3.0.x.x_Maintenance. Especially so because many webhosts now require PHP 8.0 or 8.1.

**Comment by TheCartpenter** [comment]
I'm glad you ask. Since communication is eminent in here in order to move forward with these bug-fixes, let's start with these commits. Then, we'll see how things goes for the next release.

**Comment by mhcwebdesign** [comment]
@TheCartpenter: I doubt anybody will  be able to go through all your pull requests, I certainly won't, there are simply too many of them, and many of them aren't even bugfixes. I strongly suggest you group them, and don't split them up into hundreds of small pull requests each day. Use a proper git client for this, such as  SmartGit or others.

**Reply by TheCartpenter** [reply]
```@TheCartpenter: I doubt anybody will be able to go through all your pull requests, I certainly won't, there are simply too many of them, and many of them aren't even bugfixes. I strongly suggest you group them, and don't split them up into hundreds of small pull requests each day. Use a proper git client for this, such as SmartGit or others.```

Well, what a shame. To think that people who are still waiting and more that are going to wait for these bugfixes to be integrated will be because of you and from the people who do have the actual access to integrate these. One thing is for sure, if I'd had those access, it'd be in no matter the quantity since it's the quality that matters. As for the PHP 8.x, the master branch will take the lead on anyhow.

**Comment by mhcwebdesign** [comment]
You haven't answered my question: Is there nay reason why you won't make life easier for others, by combining your pull-requests using a proper git client?

**Comment by TheCartpenter** [comment]
That reason has already been explained to you multiple times on this repository. Not only you're not taking initiative to add those pull requests but, now, obvious that you're not reading the replies either. One thing's for sure, people do have a lot of patience to wait for something good to happen when things keeps getting delayed like that.

**Comment by ADDCreative** [comment]
@TheCartpenter Please consider others and stop spamming pull requests. You need to combine them so that they can easily be reviewed and tested before merging.

For example the 3 following should be as one pull request. That way it can reviewed and even checkouted and tested.
https://github.com/opencart/opencart/pull/10820
https://github.com/opencart/opencart/pull/10821
https://github.com/opencart/opencart/pull/10822

Doing it in three separate requests makes it very difficult for the maintainers to see the whole change. Multiply this by the hundreds submitted each day and it becomes an impossible task.

**Comment by TheCartpenter** [comment]
The reason why it makes it difficult for the maintainers is, again, based on the above statement from my previous reply in regards of waiting too long before taking initiative from the people who are able to merge these pull requests into the Maintenance Branch. Besides, how can we explain that a single guy within 6-7 hours is able to do all this in a flesh but the only people who are complaining about it are the people who are unable to follow through each posts.

The impossible task, on this end, then becomes possible since I was able to submit all these changes within approximately a quarter of the day just by using the web version of Github, and without the need to use any additional tools to achieve this. Therefore, since Github already provides the ability for developers to submit their pull requests from their website, and on this repository, there are no valid reasons whatsoever where a developer should feel the need to install additional tools or software on his computer especially with the fact that passwords are required even though our Github account is linked to the branch as opposed to the web version. So to add, Github does neither require to developers for software installations. They do 'recommend' it, but isn't in any obligations to do so nor should the people be pushed frequently in order to achieve the same goals that can be accomplished on the web already. The rest goes to the delays being put on the assignee that needs to keep track of the pull requests being assigned.

Again, if it'd be up to me, rather than wasting time arguing about this, it'd already been done including the pull requests from other Github users that can be merged to improve the platform flexibility. So far, I fail to see this happening due to the mass delays being involved in the process.

**Comment by ADDCreative** [comment]
@TheCartpenter It seems clear that the maintainers don't have the time or resources, so it makes no sense to add to their workload. You will just end up wasting your own time as well as theirs.

**Comment by TheCartpenter** [comment]
AFAIK, no resources was required to create these commits. If I had already wasted time in anyway to send pull requests to resolve bug-fixes, especially the ones from yesterday, then I guess you'd be voluntarily addressing the Github community on that (which I don't, just to be clear) based on the fact that Github users can still use the web version to submit their code changes.

Therefore, the true and only time wasted is in here to actually argue about this instead of jumping ahead from the momentum level. The only reason why all these pull requests remains in there, as part to what's been said on the above, is that the actual people who are eligible to do it are not dedicated to take care of it.

**Reply by condor2** [reply]
@mhcwebdesign the branch is not ready for 3.0.3.9...
Some bugs not fixed....

If I had merge permission I will updated Jquery to 3.6.0, Summernote to 0.8.20, bootsrap to 3.4.1....

All in one :)
https://github.com/opencart/opencart/pull/10823

**Reply by TheCartpenter** [reply]
> @mhcwebdesign the branch is not ready for 3.0.3.9... Some bugs not fixed....
> 
> If I had merge permission I will updated Jquery to 3.6.0, Summernote to 0.8.20, bootsrap to 3.4.1....
> 
> All in one :) #10823

Both, TWIGs and JS, needs implementing.

**Comment by ADDCreative** [comment]
Looks like there is a new respostity for the 3.0.x.x_Maintenance.
https://github.com/opencart/opencart/pull/10817#issuecomment-1133262798

Can someone explained what is planned for this. Will there ever be a another release for the 3.x branch? Will it be just critical bug fixes or proper maintenance (e.g. updating jQuery as suggested by @condor2)?

**Comment by TheCartpenter** [comment]
@ADDCreative: One day a time. At least, those days can finally be numbered.

**Comment by pemaier** [comment]
May can someone tell me what is the difference between opencart/opencart 3.0.xx_maintenance and opencart/opencart-3
Which repository is to use for e.g. testing?
